browser/_layouts/schema.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="{{ site.lang | default: "en-US" }}">
+  <head>
+    <meta charset="UTF-8">
+
+{% seo %}
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="theme-color" content="#157878">
+    <link rel="stylesheet" href="{{ '/assets/css/style2.css?v=' | append: site.github.build_revision | relative_url }}">
+  </head>
+  <body>
+    <div id="container">
+      <nav id="sidebar">
+        <img id="logo" src="{{ '/browser/assets/images/rubinobs.png' | relative_url }}"/>
+        <ul>
+          <li><a href=".">Schema Browser</a></li>
+          <ul>
+            <li><a href="">{{ page.title }}</a></li>
```

Comment:
Include the hash or tag/version somewhere?

Reply:
Yes, that is a (the most?) serious drawback of this approach: one version (master/main) only.  Something based on lsst-the-docs would be superior in this regard.  This solution is proposed as a low effort stopgap.